.github/actions/setup-node/action.yaml

```
@@ -0,0 +1,29 @@
+name: Setup Node Only
+description: Lightweight Node.js setup with modern pnpm caching (2025)
+
+inputs:
+  github_token:
+    description: GitHub token for authentication
+    required: false
+
+runs:
+  using: "composite"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)